docs/sources/datasources/alertmanager.md

```diff
@@ -0,0 +1,38 @@
++++
+title = "Alertmanager"
+description = "Guide for using Alertmanager in Grafana"
+keywords = ["grafana", "prometheus", "guide"]
+aliases = ["/docs/grafana/latest/features/datasources/alertmanager"]
+weight = 1300
++++
+
+# Alertmanager data source
+
+Grafana includes built-in support for Prometheus Alertmanager. Add it as a datasource and you will be able to use the [Grafana alerting UI](https://grafana.com/docs/grafana/latest/alerting/) to manage silences, contact points and notification policies. A dropdown will appear in these pages to allow you to switch between Grafana and any configured alertmanager data sources This data source cannot return graph data and is not available in panels.
```

Comment:
```suggestion
Grafana includes built-in support for Prometheus Alertmanager. Once you add it as a data source, you can use the [Grafana unified alerting UI](https://grafana.com/docs/grafana/latest/alerting/) to manage silences, contact points as well as notification policies. A drop down option in these pages allows you to switch between Grafana and any configured Alertmanager data sources 

The Alertmanager data source cannot return graph data and is not available in panels.
```

Reply:
Are we publically calling this "unified" alerting? I thought "unified" is an internal designation

Reply:
updated, but as per slack discussion, w/o "unified" word